apps/server/src/modules/authorization/authorization-context.builder.ts

```diff
@@ -1,6 +1,7 @@
 import { Permission } from '@shared/domain';
 import { AuthorizationContext, Action } from './types';
 
+// need to move to shared
```

Comment:
Warum?

Reply:
Ich nehme den Kommentar raus.

Reply:
Ich pushe die Änderungen dann gebündelt später.

Reply:
Spontan würd ich sagen: Wenn etwas nur mit Authorisierung zu tun hat, dann sollte es im AuthorizationModule liegen. Das gilt für den AuthorizationContext, nehm ich an. Wenn Action auch für andere Sachen als Authorisierung benutzt wird, dann fänd ichs logisch, wenn es in shared liegt.